.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "plugins": ["import"],
+  "extends": ["react-app", "react-app/jest", "prettier"],
+  "rules": {
+    "react-hooks/exhaustive-deps": "error",
+    "@typescript-eslint/no-redeclare": "error",
+    "@typescript-eslint/no-unused-vars": "error",
+    "import/order": [
+      "warn",
```

Comment:
Warn to start since this will cause builds to break. We'll add a followup PR will update all the imports and flip this to an error.